test/unit/ui/camera.test.js

```diff
@@ -673,7 +673,7 @@ test('camera', (t) => {
         t.test('zooms around a point', (t) => {
             const camera = createCamera();
             camera.easeTo({around: [100, 0], zoom: 3, duration: 0});
-            t.deepEqual(fixedLngLat(camera.getCenter()), fixedLngLat({lng: 87.5, lat: 0}));
+            t.deepEqual(fixedLngLat(camera.getCenter()), fixedLngLat({lng: 92.946845206, lat: 0}));
```

Comment:
This change looks quite big. Do you know what could be causing it?

Reply:
Not yet but I'll switch that one to a FIXME instead so we can investigate further.